clickhouse-admin/types/src/lib.rs

```diff
@@ -1007,7 +1008,7 @@ pub struct DistributedDdlQueue {
     /// Exception message
     pub exception_text: String,
     /// Query finish time
-    pub query_finish_time: String,
+    pub query_finish_time: DateTime<Utc>,
     /// Duration of query execution (in milliseconds)
     pub query_duration_ms: String,
```

Comment:
Looks like this is a `UInt64`, is it worth parsing into a stronger type during deserialization? Something like a `Duration`, or at least `u64`?

Reply:
It's a string with a number in it :( I initially had it as a u64 and it didn't parse, but let me try with `Duration`!

Reply:
Ah, this is because we're asking for JSON, and by default ClickHouse quotes 64-bit numbers. That causes `serde` to think it's a string, because it's written like '"1"`, rather than just `1`. You can set the [`output_format_json_quote_64bit_integers`](https://clickhouse.com/docs/en/operations/settings/formats#output_format_json_quote_64bit_integers) setting to ask it not to that.

Reply:
Looks like Duration doesn't accept strings either :(

Reply:
See my earlier comment. You can ask CH not to quote big integers, so that `serde` can parse it just fine.

Reply:
ugh sorry, GitHub hid that comment from me, I see it now thanks!

Reply:
No worries, I figured :) Just didn't want you to lose track of it. I think that will work, but it's honestly up to you if you want to apply it. I don't have context for what this all will be used for!

Reply:
That worked, thanks! It's definitely better with strong types. Wasn't able to get `Duration` to parse but at least it's `u64` now. We're going to use this endpoint (and others) for long term monitoring and debugging of the ClickHouse cluster.